torch/ao/nn/quantized/modules/activation.py

```diff
@@ -271,6 +271,11 @@ def from_float(cls, mod):
         qprelu = cls(float(scale), int(zero_point), mod.num_parameters)
         float_wt = mod.weight.float()
         observer = mod.qconfig.weight()
+        observer(float_wt.unsqueeze(-1))
```

Comment:
why do we need to unsqueeze?

Reply:
because if you try to use a per_channel weight observer on it, then it will throw an error because y = torch.flatten(y, start_dim=1) doesn't work when dim=1 is out of bounds. Its why (https://github.com/pytorch/pytorch/pull/103455/commits/9c6927f1ef9b847b26ecee9575227d27491c7662) didn't work.

Reply:
does this mean this layer does not support per channel weight quantization? I think in that case we should fix the place where people requested the per channel weight quantization, instead of here